ovssubnet/registry/registry.go

```diff
@@ -230,6 +238,11 @@ func (sub *EtcdSubnetRegistry) GetContainerNetwork() (string, error) {
 	return resp.Node.Value, err
 }
 
+func (sub *EtcdSubnetRegistry) GetServicesNetwork() (string, error) {
+	// FIXME
+	return "172.30.0.0/16", nil
```

Comment:
That's only available on the master (where it can be overridden with a command-line argument). We need it on the minions.